lib/index.js

```diff
@@ -2625,3 +2625,4 @@ var powerbi;
         })(utils = extensibility.utils || (extensibility.utils = {}));
     })(extensibility = powerbi.extensibility || (powerbi.extensibility = {}));
 })(powerbi || (powerbi = {}));
+//# sourceMappingURL=index.js.map
```

Comment:
You need to manually remove this auto-generated line because it causes errors in custom visuals test runiing.
We are looking for issue to stop generation of this line. As quick workaround just remove it.